test/DeltaTriplesTest.cpp

```diff
@@ -6,50 +6,17 @@
 
 #include <gtest/gtest.h>
 
-#include "./util/GTestHelpers.h"
-#include "./util/IndexTestHelpers.h"
+#include "DeltaTriplesTestHelpers.h"
 #include "absl/strings/str_split.h"
 #include "engine/ExportQueryExecutionTrees.h"
 #include "index/DeltaTriples.h"
 #include "index/IndexImpl.h"
 #include "index/Permutation.h"
 #include "parser/RdfParser.h"
+#include "util/GTestHelpers.h"
```

Comment:
```suggestion
#include "./util/GTestHelpers.h"
```
(Same for the line below)
Unfortunately we don't yet have absolute include paths within the `test/` directory.